src/libraries/System.Drawing.Common/src/System/Drawing/Image.Unix.cs

```diff
@@ -150,10 +150,10 @@ public Image GetThumbnailImage(int thumbWidth, int thumbHeight, Image.GetThumbna
             return ThumbNail;
```

Comment:
On row 129 ` Image.GetThumbnailImageAbort callback, IntPtr callbackData` are not used for calling function ` Gdip.GdipDrawImageRectRectI( ... )` (null passed instead) not sure if this is a bug or intentional, if it is intention then `callback` should be nullable

Reply:
The corresponding windows implementation does use the argument, however it's not clear looking at just the managed code whether it can be nullable. I'd err towards keeping non-nullable for now.